Taskfile.yml

```diff
@@ -132,7 +133,13 @@ tasks:
     desc: "Fill `test` database with data for experiments"
     cmds:
       - bin/task{{exeExt}} -d integration env-data
-
+  
+  env-repl:
```

Comment:
I think the reason for creating this is to build fewer images so it's quicker? And you would want independence from `SERVICES` for testing oplog and change streams?

Reply:
Yes. I also think it's best to separate it the common development environment to avoid confusion and to not slow things down.